### PR TITLE
Add "no …" filter in dashboard

### LINKED ISSUE #3606: Sort in Activities Overview for no State / no Responsible
In der Übersicht aller Blöcke ist es aktuell nur möglich, nach Blöcken zu filtern, denen ein Status zugeordnet ist. Es ist nicht möglich, nach Blöcken zu ordnen, denen noch kein Status zugewiesen worden ist.

### PR DESCRIPTION
**Pull request by manuelmeister**
Bildschirmfoto 2023-06-29 um 18 06 49 Bildschirmfoto 2023-06-29 um 19 28 10

Closes #3606

### CODE REVIEW

File: frontend/src/views/camp/Dashboard.vue [hunk 1]
```diff
@@ -286,7 +292,9 @@ export default {
             this.filter.progressLabel.length === 0 ||
             this.filter.progressLabel?.includes(
               scheduleEntry.activity().progressLabel?.()._meta.self
-            ))
+            ) ||
+            (scheduleEntry.activity().progressLabel === null &&
+              this.filter.progressLabel?.includes('none')))
```
Comment (pmattmann) [inline frontend/src/views/camp/Dashboard.vue]: Couldn't you just do the following:
```
            this.filter.progressLabel?.includes(
              scheduleEntry.activity().progressLabel?.()._meta.self ?? 'none'
            ) 
```
means, if ProgressLabel == null, then use dummy-id 'none'

Comment (manuelmeister) [reply frontend/src/views/camp/Dashboard.vue]: Nice, had a moment to wrap my head around

**Review by carlobeltrame — COMMENTED** [review]
As per https://github.com/ecamp/ecamp3/issues/3606#issuecomment-1612153494, could you maybe do the same for activityResponsibles?

**Comment by manuelmeister** [comment]
@carlobeltrame working on it, but it required more logic, as a "No responsibles" is an exclusive none instead of a "normal" none.

**Review by pmattmann — APPROVED** [review]

**Review by carlobeltrame — APPROVED** [review]
The filters are currently not keyboard accessible. We should fix that (in a separate PR).

File: frontend/src/components/dashboard/SelectFilter.vue [hunk 2]
```diff
@@ -24,14 +24,19 @@
...
-          <slot name="item" v-bind="{ item, self }">{{ item.text }} </slot>
+          <slot name="item" v-bind="{ item, self }">{{ item.text }}</slot>
         </v-list-item-title>
-        <v-list-item-action v-if="multiple">
+        <v-list-item-action v-if="multiple && !item.exclusiveNone">
           <v-checkbox v-model="item.selected" dense />
         </v-list-item-action>
+        <v-list-item-action v-if="item.exclusiveNone">
+          <v-radio-group v-model="item.selected">
+            <v-radio dense :value="true" />
```
Comment (usu) [inline frontend/src/components/dashboard/SelectFilter.vue:37]: At first glance, I was confused why this was a radiobox for responsibles, but a checkbox for status. But then I recalled the discussion we had with OR and AND selection 🙄 

Let's see how the users cope with it and whether we receive any feedback on it.

**Review by usu — APPROVED** [review]

**Pull request merged**